lemur/factory.py

```diff
@@ -161,12 +179,12 @@ def configure_extensions(app):
         )
 
     if app.config["CORS"]:
-        app.config["CORS_HEADERS"] = "Content-Type"
         cors.init_app(
             app,
             resources=r"/api/*",
-            headers="Content-Type",
-            origin="*",
+            origins=app.config.get("CORS_ORIGIN", ["*"]),
```

Comment:
For backwards compatibility, we likely want to maintain the old defaults while allowing for overrides. I think we can do this by setting if not present for the CORS_ORIGINS and CORS_ALLOW_HEADERS config options. For example,
```
if "CORS_ORIGINS" not in app.config:
  app.config["CORS_ORIGINS"] = "*"
```
Then we can remove the origin and headers kwargs and let flask_cors pick up the set options